app/views/home/index.html.haml

```diff
@@ -75,7 +75,7 @@
         .home-services-buttons.container
           .insert-sample-text-button{'data-action': 'click->sample-text#annotator_recommender', 'data-sample-text': t("sample_text")}
             %div
-              Insert sample text
+              = t('home.insert_sample_text')
```

Comment:
insert_sample_text already exists in annotator and recommender, 
replace this with:
= t('insert_sample_text')